Description: Change AboutBox logo
 Change the logo in the About dialog from the default GNOME 1-esque shell
 to system-file-manager, bit more pleasing on the eye and works with the
 theme
 .
 nautilus (1:3.4.2-2solusos2) two; urgency=low
 .
   * Changed about logo to system-file-manager (more pleasing than a shell..)
Author: Ikey Doherty <ikey@solusos.com>

---
Reviewed-By: Ikey Doherty <ikey@solusos.com>
Last-Update: <2012-08-06>

--- nautilus-3.4.2.orig/src/nautilus-window-menus.c
+++ nautilus-3.4.2/src/nautilus-window-menus.c
@@ -398,7 +398,7 @@ action_about_nautilus_callback (GtkActio
 				 * box to give credit to the translator(s).
 				 */
 			      "translator-credits", _("translator-credits"),
-			      "logo-icon-name", "nautilus",
+			      "logo-icon-name", "system-file-manager",
 			      "website", "http://live.gnome.org/Nautilus",
 			      "website-label", _("Nautilus Web Site"),
 			      NULL);
